src/components/Button/Button.js

```diff
@@ -15,7 +15,13 @@ const Btn = (props) => {
       style={{...styles.button, ...props.styles, ...disabledStyle}}
       onPress={props.onPress}
       {...props}>
-      <Text style={{...styles.buttonText, ...props.textStyling}}>{props.children}</Text>
+      <Text
+        style={[
+          {...styles.buttonText, ...props.textStyling},
+          disable ? styles.buttonTextDisabled : {}
+        ]}>
+        {props.children}
+      </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `disable` prop is used in the Text component to apply a different style when the button is disabled. However, it seems like this prop is not defined or passed to the Button component. This could lead to an error or unexpected behavior. Please ensure that the `disable` prop is correctly defined and passed to the Button component.

```diff
-          disable ? styles.buttonTextDisabled : {}
+          props.disable ? styles.buttonTextDisabled : {}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->